.github/workflows/build.yml

```diff
@@ -19,10 +19,10 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Using Node.js 12.x
+      - name: Using Node.js 14.x
         uses: actions/setup-node@v1
         with:
-          node-version: 12.x
+          node-version: 14.x
```

Comment:
Do we want to update this in other GitHub workflows too?

Reply:
We should!

Reply:
Do we want to update them in this PR :)